src/args.c

```diff
@@ -18,18 +18,26 @@
  * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
  */
 
+#include "config.h"
 #include "args.h"
 #include "charset.h"
 #include "list.h"
 #include "options.h"
 
 #include <stdio.h>
 #include <string.h>
-#include <strings.h>
 #include <stdlib.h>
 #include <ctype.h>
 #include <assert.h>
+
+#ifdef __unix__
 #include <sys/param.h>
+#include <strings.h>
+#endif
+
+#if !defined(HAVE_STRCASECMP) && defined(HAVE_STRICMP)
+#define strcasecmp _stricmp
+#endif
```

Comment:
I don't like littering copies of this everywhere. Since this is only about one single function, maybe move it to some header like `strcasecmp.h`?

Reply:
Will do. Wasn't sure whether that will be accepted considering that this project seems to be gcc-focused.